src/workflows/airqo_etl_utils/data_validator.py

```
@@ -5,7 +5,7 @@
 
 from airqo_etl_utils.bigquery_api import BigQueryApi
 from airqo_etl_utils.constants import ColumnDataType
-from typing import Any
+from typing import Any, Optional, Union
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import

(F401)